src/features/SendAssets/components/SendAssetsForm.tsx

```diff
@@ -200,6 +201,14 @@ export const SendAssetsForm = ({ txConfig, onComplete, protectTxButton }: ISendF
   const [isResolvingName, setIsResolvingDomain] = useState(false); // Used to indicate recipient-address is ENS name that is currently attempting to be resolved.
   const [fetchedNonce, setFetchedNonce] = useState(0);
   const [isSendMax, toggleIsSendMax] = useState(false);
+  const [selectedNetwork, setSelectedNetwork] = useState<Network>(getNetworkById(DEFAULT_NETWORK));
```

Comment:
There shouldn't be a need to add more state for this. It is already present in `values`.

Reply:
I investigated about lifting Formik state to outside of the form or to make validation schema dynamic but I couldn't find an easier way other than adding a new state. As you have noticed, I need to update the `Schema` according to `network field changes`. I will appreciate if you may suggest something regarding that issue.

Reply:
I found out the solution by using `when` in yup validation.

https://github.com/formium/formik/issues/1228#issuecomment-453747566